### ISSUE TITLE
Download query cmf

### ISSUE THREAD

**Issue by sanni85** [LMFDB/lmfdb#3284]
If one downloads the results of a cmf search, for example as a sage file, one gets this (extract)

```
# Classical modular forms downloaded from the LMFDB on 30 August 2019.
# Query "{'level': 12}" returned 151 forms.

# Below are two lists, one called labels, and one called data (in matching order).
# Each entry in the data list has the form:
#    [N=level, k=weight, dim, N*k^2, defining polynomial, number field label, CM discriminants, RM discriminants, first few traces]
```

and here a line of the second list

[12, 3, 1, 0.326976317232, [0, 1], "1.1.1.1", [-3], [], [0, -3, 0, 2]],

The 4th field of each entry should be N*k^2 but I think it is the analytic conductor. 

Moreover, "first few traces" means always traces of a2, a3, a5 and a7, so it should say that.

**Comment by assaferan** [comment]
I will be solving this

**Comment by assaferan** [comment]
solved in pull request #3400
